rust/src/rfb/rfb.rs

```diff
@@ -194,6 +194,7 @@ impl RFBState {
                                 current,
                                 consumed as i64,
                                 RFBFrameType::Pdu as u8,
+                                None,
```

Comment:
@satta should we set tx_id for RFB frames ?

Reply:
See above